### PR TITLE
Add conda_mirror_ng

### PR DESCRIPTION
**Pull request by xhochy**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by xhochy** [comment]
@conda-forge/staged-recipes This is ready for review/merge.

File: recipes/conda_mirror_ng/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  entry_points:
+    - conda-mirror-ng = conda_mirror_ng.conda_mirror:cli
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3
+    - pyyaml
+    - requests
```
Comment (jakirkham) [inline recipes/conda_mirror_ng/meta.yaml]: Are `pyyaml` and `requests` needed to build the package?

File: recipes/conda_mirror_ng/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    - setuptools_scm
```
Comment (jakirkham) [inline recipes/conda_mirror_ng/meta.yaml]: Should we add `setuptools` to `host`?

File: recipes/conda_mirror_ng/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  run:
+    - python >=3
+    - setuptools
```
Comment (jakirkham) [inline recipes/conda_mirror_ng/meta.yaml]: Is `setuptools` needed at runtime?

**Comment by scopatz** [comment]
I think we should hold off on this until we figure out where the real community fork should live.

**Comment by scopatz** [comment]
Ok the regro version 0.8.1 has been released, feel free to update this!

**Comment by xhochy** [comment]
Closing this as there is already https://github.com/conda-forge/conda-mirror-feedstock

**Comment by xhochy** [comment]
@jakirkham still FYI: `setuptools_scm` is used to determine the version at build time. This is then persisted into the package. At runtime, it uses `pkg_resources` (part of `setuptools`) to provide `__version__` at runtime.

**Comment by chrisburr** [comment]
Should `setuptools_scm` use `run_exports` to ensure `setuptools` is available at runtime?

**Comment by xhochy** [comment]
> Should `setuptools_scm` use `run_exports` to ensure `setuptools` is available at runtime?

No, the use of `pkg_resources` for `__version__` is independent of `setuptools_scm`.

**Pull request closed without merging**